studio/src/aem/aem.js

```diff
@@ -248,6 +254,16 @@ class AEM {
         }
     }
 
+    async pollUpdatedFragment(oldFragment) {
```

Comment:
👍

Reply:
I wonder if there would be a case where etag will not change because nothing really has changed. In this case It may block the UI indefinitely. Should we limit to first 10 attempt. and wait 500ms between every attempt.
Values can be changed as you see fit.